tools/get-deps/main.go

```diff
@@ -199,12 +199,25 @@ func filterPkg(deps []string) []string {
 	var depList []string
 	dpList := make(map[string]bool)
 
-	reLF  := regexp.MustCompile("lfedge/.*")
-	rePkg := regexp.MustCompile("lfedge/(?:eve-)?(.*):.*")
+	reLF := regexp.MustCompile("lfedge/.*")
+	rePkg := regexp.MustCompile("lfedge/(?:eve-)?(.*):(?P<version>.*)")
 	for _, s := range deps {
 		// We are just interested on packages from lfegde (those that we
 		// published)
 		if reLF.MatchString(s) {
+			matches := rePkg.FindStringSubmatch(s)
```

Comment:
I am not sure I get what this is doing. I _think_ that it is:

* before: if it matched `lfedge/eve-:.*` then assume that it is `pkg/<everything between eve-and tag :>`
* after: if it matched `lfedge/eve-:<sha256>` then assume that it is `pkg/<everything between eve-and tag :>`

Is that it?

If so:

1. Do you mind adding a README to this? You actually figured out how this works, it is good to capture it. Something that just describes what this does (in a sentence or two) and more importantly, by what logic it filters (the important part).
2. Is that solid enough? I can see other things following that path that do not come out of here.
3. Should we add a check for the path? Meaning, instead of (or in addition to), "if it is `lfedge/eve-.*:<sha256>`", maybe also check if `pkg/$1` exists, and, if not, clearly it doesn't belong as part of get-deps, so skip it?

Reply:
> 3\. Should we add a check for the path? Meaning, instead of (or in addition to), "if it is `lfedge/eve-.*:<sha256>`", maybe also check if `pkg/$1` exists, and, if not, clearly it doesn't belong as part of get-deps, so skip it?

yes, this is what I wrote above. Maybe the easiest way. 

> Do you mind adding a README to this
I would, but not now. I fixed this one because it was a show stopper for my 'eve visibility' task

Reply:
Aw, no big deal. Two paragraphs. You know it never will get done otherwise. Who is going to take time to write it later?

> yes, this is what I wrote above. Maybe the easiest way

Go for it.